.devcontainer/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See LICENSE file in the project root for license information.
 #-------------------------------------------------------------------------------------------------------------
 
-FROM mcr.microsoft.com/vscode/devcontainers/python:0-3.10
+FROM mcr.microsoft.com/vscode/devcontainers/python:3.10-bookworm

```

Comment:
Actually, if you specify only the Python version (`:3.10`):
```docker
FROM mcr.microsoft.com/vscode/devcontainers/python:3.10
```
then it will use the latest Debian image as the base (currently Bookworm, but it will get updated automatically with the new version).